turbinia/processors/partitions.py

```diff
@@ -46,11 +46,28 @@ def Enumerate(evidence):
     scanner = volume_scanner.VolumeScanner(mediator=mediator)
     path_specs = scanner.GetBasePathSpecs(evidence.local_path)
   except dfvfs_errors.ScannerError as e:
-    raise e
+    raise TurbiniaException(
+        'Could not enumerate partitions [{0!s}]: {1!s}'.format(
+            evidence.local_path, e))
 
   return path_specs
 
 
+def GetPartitionEncryptionType(path_spec):
+  """Checks a partition for encryption.
+
+  Args:
+    path_spec (dfVFS.path_spec): Partition path_spec.
+
+  Returns:
+    String representing the type of encryption, or None.
+  """
+  encryption_type = None
```

Comment:
Thanks for moving this into this module.  Could we also either late load the dfvfs module here, or late load the partitions module in the evidence module so that it won't get loaded when the client generates evidence for procssing?